src/Microsoft.Win32.Registry/src/Microsoft.Win32.Registry.csproj

```diff
@@ -31,7 +31,7 @@
     <Compile Include="Microsoft\Win32\SafeHandles\SafeRegistryHandle.cs" />
     <Compile Include="System\Security\AccessControl\RegistryRights.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp' AND '$(TargetsWindows)' == 'true'">
+  <ItemGroup Condition="('$(TargetGroup)' == 'netcoreapp' or '$(TargetGroup)' == 'uap') AND '$(TargetsWindows)' == 'true'">
```

Comment:
Yes my preference is to generally be inclusive instead of exclusive but we should keep the conditions within reason. I wonder if we should set some other property in here that means TargetsCore and use that in the various places.